src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: État de partage
+i18nReady: true
+setup: |
+  import UIFrameworkTabs from '~/components/tabs/UIFrameworkTabs.astro'
+  import LoopingVideo from '~/components/LoopingVideo.astro'
+  import JavascriptFlavorTabs from '~/components/tabs/JavascriptFlavorTabs.astro'
+---
+
+Lors de la construction d'un site Web Astro [architecture Islands / hydratation partielle](/fr/concepts/islands/), Vous avez peut-être rencontré ce problème: **Je veux partager l'état entre mes composants.**
+
+Les frameworks d'interface utilisateur comme React ou Vue peuvent encourager ["context" providers](https://reactjs.org/docs/context.html). Mais quand les [composants partiellement hydratés](/fr/core-concepts/framework-components/#hydrating-interactive-components) dans Astro ou Markdown, vous ne pouvez pas utiliser ces emballages de contexte..
+
+Astro recommande une solution différente pour le stockage partagé côté client: [**Nano Stores**](https://github.com/nanostores/nanostores).
+
+## Pourquoi Nano Stores?
+
+La librairie [Nano Stores](https://github.com/nanostores/nanostores) vous permet de créer des magasins avec lesquels n'importe quel composant peut interagir. Nous recommandons Nano Stores car
```

Comment:
I don't think we should translate "store" as I've never seen it translated, what do you think @thomasbnt ?

Reply:
_Nano Stores_ is a name, I never translate names UNLESS the author has already done so.

Reply:
Makes sense. What about "Svelte Stores"? This isn't a library, just "Svelte's taste of stores". My suggestion was to translate it to "les Stores de Svelte".

Reply:
_Oh don't hesitate to mention me if you want I see your comment_ 😆

_les Stores de Svelte_ seems to me correct! ✔️

Reply:
Following @MoustaphaDev’s suggestion this would be:

```suggestion
La librairie [Nano Stores](https://github.com/nanostores/nanostores) vous permet de créer des stores avec lesquels n'importe quel composant peut interagir. Nous recommandons Nano Stores car
```

Reply:
> Following @MoustaphaDev’s suggestion this would be:

Right !